src/Model/PhasorDynamics/Branch/Branch.cpp

```diff
@@ -2,48 +2,49 @@
  * @file Branch.hpp
  * @author Slaven Peles (peless@ornl.gov)
  * @brief Definition of a phasor dynamics branch model.
- * 
+ *
  * The model uses Cartesian coordinates.
- * 
+ *
  */
 
-#include <iostream>
+#include "Branch.hpp"

```

Comment:
Is this the desired behavior? `#include "*hpp"` before system includes, e.g., (`#include <iostream>`). I would intuitively expect the local includes to be last.